### PR TITLE
user list

### PR DESCRIPTION
**Pull request by KunegundaKosek**
Improving the user list

### CODE REVIEW

File: src/pages/UserList/index.tsx [hunk 1]
```diff
@@ -16,25 +19,31 @@ import { EmployeeContext } from "../../Context/AppContext";
 // };
 
 const UserList = () => {
-  const { employees, deleteEmployee } = useContext(EmployeeContext);
+  const { employees } = useContext(AppContext);
 
   return (
-    <div>
+    <>
       <h1>USER LIST</h1>
```
Comment (szu-szy) [inline src/pages/UserList/index.tsx]: tag for section headline should be <h2>

File: src/pages/UserList/index.tsx [hunk 2]
```diff
@@ -16,25 +19,31 @@ import { EmployeeContext } from "../../Context/AppContext";
...
       <ul>
         {employees.map((employee) => (
           <li key={employee.id}>
-            <h2>
```
Comment (szu-szy) [inline src/pages/UserList/index.tsx:27]: then h2 should be changed to h3

**Review by szu-szy — CHANGES_REQUESTED** [review]

**Review by szu-szy — CHANGES_REQUESTED** [review]
still have comments to change

**Review by szu-szy — CHANGES_REQUESTED** [review]
still comments

**Comment by KunegundaKosek** [comment]
I still don't know what comments you're talking about

**Review by szu-szy — APPROVED** [review]

**Pull request merged**